src/oph/ehoks/oppija/share_handler.clj

```diff
@@ -46,7 +115,7 @@
       :return (rest/response [oppija-schema/Jakolinkki])
       :summary "Jaettuun moduuliin liitettyjen jakolinkkien haku"
       :path-params [uuid :- String]
-      (let [jakolinkit (db/select-shared-module-links uuid)]
+      (let [jakolinkit (fetch-shared-module-data uuid)]
```

Comment:
Tässä ei taideta tarvita näin paljon dataa, frontti ei tarvitse tästä routesta kuin kolme kenttää share-id, voimassaolo-alku ja voimassaolo-loppu. Tämä kannattaisi irroittaa kokonaan omaksi skeemaksi mutta kommentoin siitä skeemojen kohdalla.